### DISCUSSION TITLE
Automatic filter

### DISCUSSION THREAD

**Discussion by Jokopentil** [uBlockOrigin/uBOL-home#80] [Ideas]
does ublock origin have an automatic filter, !  For example, if I'm from Indonesia, I open a website from Russia and automatically block ads without turning on the Russian filter? I request it

**Comment by MasterKia** [comment]
Duplicate of https://github.com/uBlock-LLC/uBlock/issues/957#issuecomment-77712912

> gorhill: There are all kind of complaints, including those who think it blocks too much. Given that the regional lists are of widely variable quality, I rather let the users pick explicitly for anything else than the default ones.
